### PR TITLE
Dev to Main Sync

### PR DESCRIPTION
**Pull request by kotesh-arya**
**Developer Name:** @kotesh-arya  @Shenoy07 @Aryex82 @Achintya-Chatterjee @Aanchal-Tiwari 

**PR Number(s):-** #496 #486 #498 #509 #507

**Backend changes**

- [ ] Yes
- [x] No

**Frontend Changes**

- [x] Yes
- [ ] No        

**Database changes**

- [ ] Yes
- [x] No

**Breaking changes (If your feature is breaking/missing something please mention pending tickets)**

- [x] No
- [ ] Yes:- 

**Deployment notes(Any special instructions for the deployment)**

- [x] No

**Tested in staging**

- [x] Yes
- [ ] No

 **Media to show changes**

Add `Sync Users Nicknames` button
Screenshot from 2023-09-05 12-39-21

Add `Onbarding>31D` checkbox
[Working in staging.webm](https://github.com/Real-Dev-Squad/website-dashboard/assets/55826451/b2331ee9-829f-4f16-8d13-d2866bf5b869)

### CODE REVIEW

**Review by Aryex82 — DISMISSED** [review]
Please merge this PR once the backend PR is merged: https://github.com/Real-Dev-Squad/website-backend/pull/1499

**Comment by Achintya-Chatterjee** [comment]
Don't merge this PR before all the developers verify their changes in staging and update this PR accordingly

**Comment by lakshayman** [comment]
Merging this PR. Verified from @kotesh-arya, all the backend changes are merged.

**Review by lakshayman — APPROVED** [review]

**Pull request merged**
